tests/names/pytestnamesmodule.cpp

```diff
@@ -290,8 +290,7 @@ static struct PyModuleDef moduledef = {
 #define INITERROR return
 #endif
 
-extern "C" {
-PyMODINIT_FUNC
+extern "C" PyMODINIT_FUNC
```

Comment:
Did ``PyMODINIT_FUNC`` gain an open brace (``{``)?  It is hard to tell from here.

Reply:
Before the extern "C" was around the entire function, now it's part of the declaration since only one function in the file needs to be extern "C".